### ISSUE TITLE
Kitsu Tracking Error "http 422"

### ISSUE THREAD

**Issue by Rubberxgum** [CarlosEsco/Neko#871]
### Device information
* Neko version: version 2.8.0.2
* Android phone & version: Redmi Note 10 - Android 12

## Steps to reproduce
1. Open the manhwa "Wandering Warrior of Wudang"
2. Press tracking button.
3. Select Kitsu as tracker.

### Expected behavior
 You should be able to use Kitsu to track your progress on reading the manhwa.

### Actual behavior
 It is showing "http 422" and is failing to add Kitsu as a tracker for that manhwa.

### Other details
 This error is only showing when I'm trying to use Kitsu with this specific manhwa. I have tried using Kitsu with other manga and manhwa and it is working perfectly fine.

![Screenshot_20220227-162303_Pixel Launcher.png](https://user-images.githubusercontent.com/90501519/155874849-2af512c1-d23e-4d86-9118-ceb6a9f14714.png)

**Comment by Rubberxgum** [comment]
Here's the crash logs:
` App version: 2.8.0.2 (standard, cdebd14, 141, 2022-02-23T00:28:00.609282)
Android version: 12 (SDK 31)
Android build ID: SQ1D.220205.003
Device brand: Redmi
Device manufacturer: Xiaomi
Device name: mojito
Device model: Redmi Note 10
Device product name: aosp_mojito--------- beginning of main
02-27 16:19:46.825 22444 22444 E hiyomi.mangade: Not starting debugger since process cannot load the jdwp agent.
02-27 16:19:47.291 22444 22501 E SQLiteLog: (284) automatic index on C(manga_id)
02-27 16:19:47.291 22444 22501 E SQLiteLog: (284) automatic index on R(manga_id)
02-27 16:19:47.291 22444 22501 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
02-27 16:19:54.147 22444 22500 E SQLiteLog: (284) automatic index on mangas(_id)
02-27 16:19:56.077 22444 22498 E ||NEKO  : 	├ eu.kanade.tachiyomi.source.online.handlers.PageHandler$fetchPageList$2.invokeSuspend(PageHandler.kt:44)
02-27 16:19:56.077 22444 22498 E ||NEKO  : 	└ kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:4)
02-27 16:19:56.077 22444 22498 E ||NEKO  : error processing page list 
02-27 16:19:56.077 22444 22498 E ||NEKO  : kotlinx.coroutines.JobCancellationException: StandaloneCoroutine was cancelled; job=StandaloneCoroutine{Cancelling}@99e9657
02-27 16:19:56.079 22444 22498 E ||NEKO  : 	├ eu.kanade.tachiyomi.util.system.RxCoroutineBridgeKt$runAsObservable$1$job$1.invokeSuspend(RxCoroutineBridge.kt:9)
02-27 16:19:56.079 22444 22498 E ||NEKO  : 	└ kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:4)
02-27 16:19:56.079 22444 22498 E ||NEKO  : error in coroutine bridge
02-27 16:19:56.079 22444 22498 E ||NEKO  : kotlinx.coroutines.JobCancellationException: StandaloneCoroutine was cancelled; job=StandaloneCoroutine{Cancelling}@99e9657
02-27 16:21:22.201 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:22.201 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:22.201 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:24.840 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:24.840 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:24.840 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:27.706 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:27.706 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:27.706 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:31.125 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:31.125 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:31.125 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:38.563 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:38.563 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:38.563 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:42.762 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:42.762 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:42.762 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:47.674 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:47.674 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:47.674 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 `

**Comment by Rubberxgum** [comment]
App version: 2.8.0.2 (standard, cdebd14, 141, 2022-02-23T00:28:00.609282)
Android version: 12 (SDK 31)
Android build ID: SQ1D.220205.003
Device brand: Redmi
Device manufacturer: Xiaomi
Device name: mojito
Device model: Redmi Note 10
Device product name: aosp_mojito--------- beginning of main
02-27 16:19:46.825 22444 22444 E hiyomi.mangade: Not starting debugger since process cannot load the jdwp agent.
02-27 16:19:47.291 22444 22501 E SQLiteLog: (284) automatic index on C(manga_id)
02-27 16:19:47.291 22444 22501 E SQLiteLog: (284) automatic index on R(manga_id)
02-27 16:19:47.291 22444 22501 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
02-27 16:19:54.147 22444 22500 E SQLiteLog: (284) automatic index on mangas(_id)
02-27 16:19:56.077 22444 22498 E ||NEKO  : 	├ eu.kanade.tachiyomi.source.online.handlers.PageHandler$fetchPageList$2.invokeSuspend(PageHandler.kt:44)
02-27 16:19:56.077 22444 22498 E ||NEKO  : 	└ kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:4)
02-27 16:19:56.077 22444 22498 E ||NEKO  : error processing page list 
02-27 16:19:56.077 22444 22498 E ||NEKO  : kotlinx.coroutines.JobCancellationException: StandaloneCoroutine was cancelled; job=StandaloneCoroutine{Cancelling}@99e9657
02-27 16:19:56.079 22444 22498 E ||NEKO  : 	├ eu.kanade.tachiyomi.util.system.RxCoroutineBridgeKt$runAsObservable$1$job$1.invokeSuspend(RxCoroutineBridge.kt:9)
02-27 16:19:56.079 22444 22498 E ||NEKO  : 	└ kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:4)
02-27 16:19:56.079 22444 22498 E ||NEKO  : error in coroutine bridge
02-27 16:19:56.079 22444 22498 E ||NEKO  : kotlinx.coroutines.JobCancellationException: StandaloneCoroutine was cancelled; job=StandaloneCoroutine{Cancelling}@99e9657
02-27 16:21:22.201 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:22.201 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:22.201 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:24.840 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:24.840 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:24.840 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:27.706 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:27.706 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:27.706 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:31.125 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:31.125 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:31.125 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:38.563 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:38.563 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:38.563 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:42.762 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:42.762 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:42.762 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:47.674 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:47.674 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:47.674 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422

**Comment by Rubberxgum** [comment]
Crash Logs:
App version: 2.8.0.2 (standard, cdebd14, 141, 2022-02-23T00:28:00.609282)
Android version: 12 (SDK 31)
Android build ID: SQ1D.220205.003
Device brand: Redmi
Device manufacturer: Xiaomi
Device name: mojito
Device model: Redmi Note 10
Device product name: aosp_mojito--------- beginning of main
02-27 16:19:46.825 22444 22444 E hiyomi.mangade: Not starting debugger since process cannot load the jdwp agent.
02-27 16:19:47.291 22444 22501 E SQLiteLog: (284) automatic index on C(manga_id)
02-27 16:19:47.291 22444 22501 E SQLiteLog: (284) automatic index on R(manga_id)
02-27 16:19:47.291 22444 22501 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
02-27 16:19:54.147 22444 22500 E SQLiteLog: (284) automatic index on mangas(_id)
02-27 16:19:56.077 22444 22498 E ||NEKO  : 	├ eu.kanade.tachiyomi.source.online.handlers.PageHandler$fetchPageList$2.invokeSuspend(PageHandler.kt:44)
02-27 16:19:56.077 22444 22498 E ||NEKO  : 	└ kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:4)
02-27 16:19:56.077 22444 22498 E ||NEKO  : error processing page list 
02-27 16:19:56.077 22444 22498 E ||NEKO  : kotlinx.coroutines.JobCancellationException: StandaloneCoroutine was cancelled; job=StandaloneCoroutine{Cancelling}@99e9657
02-27 16:19:56.079 22444 22498 E ||NEKO  : 	├ eu.kanade.tachiyomi.util.system.RxCoroutineBridgeKt$runAsObservable$1$job$1.invokeSuspend(RxCoroutineBridge.kt:9)
02-27 16:19:56.079 22444 22498 E ||NEKO  : 	└ kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:4)
02-27 16:19:56.079 22444 22498 E ||NEKO  : error in coroutine bridge
02-27 16:19:56.079 22444 22498 E ||NEKO  : kotlinx.coroutines.JobCancellationException: StandaloneCoroutine was cancelled; job=StandaloneCoroutine{Cancelling}@99e9657
02-27 16:21:22.201 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:22.201 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:22.201 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:24.840 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:24.840 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:24.840 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:27.706 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:27.706 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:27.706 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:31.125 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:31.125 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:31.125 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:38.563 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:38.563 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:38.563 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:42.762 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:42.762 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:42.762 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422 
02-27 16:21:47.674 22444 22444 E ||NEKO  : 	├ eu.kanade.tachiyomi.ui.manga.MangaDetailsController.trackRefreshError(MangaDetailsController.kt:1)
02-27 16:21:47.674 22444 22444 E ||NEKO  : 	└ eu.kanade.tachiyomi.ui.manga.MangaDetailsPresenter$trackError$1.invokeSuspend(MangaDetailsPresenter.kt:1)
02-27 16:21:47.674 22444 22444 E ||NEKO  : retrofit2.HttpException: HTTP 422

**Comment by Rubberxgum** [comment]
```
`6`
```

**Comment by Rubberxgum** [comment]
```
6
```
`6`

**Comment by CarlosEsco** [comment]
Can you try logging out, then relogging in?

**Comment by Rubberxgum** [comment]
> Can you try logging out, then relogging in?

I have tried that but it still doesn't work. The thing is Kitsu still works when I'm trying to use it for new manga but it shows error for this manhwa only.

**Comment by CarlosEsco** [comment]
Ok thanks I'll mess around

**Comment by Rubberxgum** [comment]
After update, I logged out and logged in again but the error still persist.

**Comment by Rubberxgum** [comment]
It's not a big deal though. So I won't be pursuing this issue anymore.

**Comment by CarlosEsco** [comment]
Can you try removing it from kitsu on the web browser then try adding through neko.  I had no issues adding it when I tried this morning

**Comment by Rubberxgum** [comment]
Removing it from the my Kitsu account didn't work, so I tested something different. I created a backup of my library then I wiped the data of the Neko app. After logging on my Mangadex and  tracker accounts, I tried adding "Wandering Warrior of Wudang" on Kitsu and it worked. It was named "Path of the Shaman" on Kitsu, btw. But then I removed it from Kitsu and restored my backup. After restoring my backup, the error came back again. Maybe the error has something to do with my library? 🤔
